OTKit/otkit-ticketing-desktop/token.yml

```diff
@@ -0,0 +1,27 @@
+props:
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-margin-vertical:
+    type: "size"
+    value: "48px"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-max-width:
+    type: "size"
+    value: "672px"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-background-color:
+    value: "rgba(0,0,0,0.8)"
```

Comment:
Not sure if this should live in `otkit-colors` or not, we should move it to there if this is universal.

Reply:
this value will be used exclusively for all modal backgrounds. is that enough to move it to otkit-colors?

Reply:
yes we should move it to colors, would you like to do it? Or I can do it later :P

Reply:
Either way I am going to remove this from ticketing token for now